app/src/main/java/ch/epfl/sweng/SDP/game/drawing/DrawingActivity.java

```diff
@@ -13,18 +14,17 @@
 import android.widget.LinearLayout;
 import android.widget.RelativeLayout;
 import android.widget.SeekBar;
-import android.widget.TableLayout;
-
-import ch.epfl.sweng.SDP.auth.Account;
-import ch.epfl.sweng.SDP.BaseActivity;
-import ch.epfl.sweng.SDP.shop.ShopItem;
-import ch.epfl.sweng.SDP.R;
 
 import com.google.android.gms.common.util.ArrayUtils;
 
 import java.util.LinkedList;
 import java.util.List;
 
+import ch.epfl.sweng.SDP.BaseActivity;
```

Comment:
It took me quite some time to reorder all imports. It would be nice if you could try to leave them ordered ;)

Reply:
It was not intentional, I'll fix that